hivemq-edge/src/main/java/com/hivemq/configuration/entity/adapter/SouthboundMappingEntity.java

```diff
@@ -38,7 +41,7 @@ public class SouthboundMappingEntity {
     private final @NotNull String tagName;
 
     @XmlElement(name = "fieldMapping", required = true)
-    private final @NotNull FieldMappingEntity fieldMapping;
+    private final @Nullable FieldMappingEntity fieldMapping;
```

Comment:
the constructor declares as NotNull, perhaps it should declare it as Nullable also

Reply:
`@Nullable` is useful because we can create that entity in unit test and test the validation logic.